### PR TITLE
chore: parameterized name for preset

### PR DESCRIPTION
**Pull request by amitmiran137**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -66,24 +67,24 @@ jobs:
...
-      
-      
+
+
     - name: Checkout superset repo
       uses: actions/checkout@v2
       with:
         repository: apache/incubator-superset
         path: incubator-superset
         ref: 0.38
```
Comment (MatanBobi) [inline .github/workflows/release.yml:77]: Maybe you can take this one from an environment variable? so once we want to upgrade we can just change it.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -96,7 +96,7 @@ prepareSupersetBuild:
     - sed -i -e '/^RUN mkdir -p \/app\/superset-frontend.*/a COPY ./superset-frontend/.npmrc /app/superset-frontend/.npmrc' "./Dockerfile"
     - cd superset-frontend
     - node ../../scripts/addDependencies.js
-    - node ../../scripts/generateMafPreset.js
+    - node ../../scripts/generatePreset.js Nielsen
```
Comment (MatanBobi) [inline .gitlab-ci.yml:99]: Is that really working? In the code you're not pulling the args, you're using env variable..

**Review by MatanBobi — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -4,13 +4,16 @@ on:
...
 
 jobs:
   main:
     name: Inject plugins into superset and deploy docker image
 
     runs-on: ubuntu-latest
-
+    env:
+      SUPERSET_VERSION: 0.38
+      PRESET_NAME: Nielsen
```
Comment (amitmiran137) [inline .github/workflows/release.yml:16]: @MatanBobi look how amazing this is

Comment (MatanBobi) [reply .github/workflows/release.yml:16]: Nice!

**Pull request merged**